content/scout/policy/remediation.md

```diff
@@ -0,0 +1,133 @@
+recommendations based on policy evaluation results. Recommendations are
+suggested actions you can take that improve policy compliance, or that add
+metadata to images which enables Docker Scout to provide better evaluation
+results and recommendations.
+
+Docker Scout provides remediation advice for the following policies:
+
+- [Outdated base images](#outdated-base-image-remediation)
+- [Supply chain attestations](#supply-chain-attestations-remediation)
+
+For images that violate policy, the recommendations focus on addressing
+compliance issues and fixing violations. For images where Docker Scout is
+unable to determine compliance, recommendations show you how to meet the
+prerequisites that ensure Docker Scout can successfully evaluate the policy.
+
+## View recommendations
+
+Recommendations appear on the policy details pages of the Docker Scout
+Dashboard. To get to this page:
+
+1. Open the [Docker Scout Dashboard](https://scout.docker.com/)
+2. Go to the **Policies** page.
+3. Select a policy in the list.
+
+The policy details page groups evaluation results into two different tabs
+depending on the policy status:
+
+- Violations
+- Compliance unknown
+
+The **Violations** tab lists images that don't comply with the selected policy.
+The **Compliance unknown** tab lists images that Docker Scout is unable to
+determine the compliance status for. When compliance is unknown, Docker Scout
+needs more information about the image.
+
+To view recommended actions for an image, hover over one of the images list to
+reveal a **View fixes** button on the right.
+
+![Remediation for policy violations](../images/remediation.png)
+
+Select the **View fixes** button to opens the remediation side panel containing
+recommended actions for your image.
+
+If there are more than one recommendations available, the primary
+recommendation displays as the **Recommended fix**. Additional recommendations
+are listed as **Quick fixes**. Quick fixes are usually actions that provide a
+temporary solution.
+
+The side panel may also contain one or more help sections related to the
+available recommendations.
+
+## Outdated base image remediation
+
+The **Outdated base images** policy checks whether the base image you use is
+up-to-date. The recommended actions displayed in the remediation side panel
+depend on how much information Docker Scout has about your image. The more
+information that's available, the better the recommendations.
+
+The following scenarios outline the different recommendations depending on the
+information available about the image.
+
+### No provenance attestations
+
+For Docker Scout to be able to evaluate this policy, you must add [provenance
+attestations](../../build/attestations/slsa-provenance.md) to your image. If
+your image doesn't have provenance attestations, compliance is undeterminable.
+
+When provenance attestations are unavailable, Docker Scout provides generic,
```

Comment:
I don't think we ended up with this flow where the user can manually select the tag that he's using. Can we remove this paragraph for the time being and I'll see to it that we get this done?

<img width="1840" alt="Screenshot 2024-01-10 at 10 34 53" src="https://github.com/docker/docs/assets/10833491/75d20b53-bd18-4f2c-a9cb-b414f4a73528">